pysat/tests/test_instrument.py

```diff
@@ -477,6 +478,8 @@ def test_data_access_by_datetime_and_name(self):
                       self.testInst.data['uts'].values[0])
 
     def test_data_access_by_datetime_slicing_and_name(self):
+        if self.testInst.name == 'testing2d':
+            pytest.skip('not supported for 2d')
```

Comment:
perhaps 'not supported for 2d' due to limited time sampling?

Reply:
could try something like...
```suggestion
            time_step = (self.testInst.index[1] - self.testInst.index[0]).value/1.E9
            offset = pds.DateOffset(seconds=10*time_step)
            start = dt.datetime(2009, 1, 1, 0, 0, 0)
            stop = start + offset
```